tests/steps/gdc_TARGET_case1.json

```diff
@@ -30,4 +30,4 @@
             "files": []
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
It would be nice (but not required) if you could revert the json files completely so the ones that only have an EOF change show up as unchanged.

Reply:
I had a difficult time trying to revert it to an older version for just certain documents so I copy/pasted older versions as the updated ones. Hopefully it shouldn't be an issue in the future